card_generator/api/v1/cards/views.py

```diff
@@ -36,7 +39,12 @@ class CardViewSet(ModelViewSet):
         detail=True,
     )
     def render(self, request, **kwargs):
-        """Generate a card from a template with the provided values."""
+        """
+        Generate a card from a template with the provided values.
+        :param request: Request object
+        :param kwargs: Extra data to pass
```

Comment:
`Extra data to pass` Does not help much the user to know what it is.

Reply:
update. what do you think with the new description @jeremi ?